src/test/integration/suite/index.ts

```diff
@@ -0,0 +1,38 @@
+import * as path from "path";
+import Mocha from "mocha";
+import glob from "glob";
+
+export function run(): Promise<void> {
+  // Create the mocha test
+  const mocha = new Mocha({
+    ui: "tdd",
+  });
+
+  const testsRoot = path.resolve(__dirname, "..");
+
+  // eslint-disable-next-line promise/param-names
+  return new Promise((c, e) => {
```

Comment:
is it callback and error?
this single latter variable names makes it harder to understand please name variables full names

Reply:
91a4e657d181ccb90e33de0a4046284f88a1f360